### PR TITLE
Buffs Machine Overload

### PR DESCRIPTION
**Pull request by PopeDaveThe3th**
## What Does This PR Do
Adds two uses to the Machine Overload Malfunction ability, bringing the total to four, comparable with Machine Override.

Increases the explosion's size to roughly that of several frag grenades duct-taped together.

## Why It's Good For The Game
Apparently, Machine Override also has four uses, thus making it more economical to use despite being objectively better than Overload.

This should make the decision over which ability to use slightly more of a thinker. 

Additionally, the explosion size has been buffed per request from Qwerty.

## Testing
Became AI, violently exploded R&D, Cargo, Command, and several Vulps.

It works very well.

## Changelog
:cl:
tweak: Increased Machine Overload uses to four. Increases blast size.
/:cl:

### CODE REVIEW

**Comment by Coolrune206** [comment]
Malf AI is very damn strong at the moment, with hitscan lasers and cyborg emitters, and more mobility options.
Can't say machine overload needs much of a buff.

**Comment by PopeDaveThe3th** [comment]
> Malf AI is very damn strong at the moment, with hitscan lasers and cyborg emitters, and more mobility options.
> 
> Can't say machine overload needs much of a buff.

While I disagree on the Overload part for the reasons I put in the PR, I can definitely understand the issues with giving borgs hitscan lasers. I'll see about maybe sending them back to projectiles later.

**Comment by Dafrek** [comment]
Increasing the explosion size of machine overload seems like it might be a good idea, but adding both of these buffs at the same time might be a bit much. Something simple like making the indication a bit more obvious might help. Stopping machine overload from just deleting the grav-gen might be good too, since having it at destroyed but repairable makes more sense.

**Comment by PopeDaveThe3th** [comment]
> Increasing the explosion size of machine overload seems like it might be a good idea, but adding both of these buffs at the same time might be a bit much. Something simple like making the indication a bit more obvious might help. Stopping machine overload from just deleting the grav-gen might be good too, since having it at destroyed but repairable makes more sense.
> 
> 

I didn't even know that could happen. I'll see about trying to fix that issue but it might be beyond my meager abilities.

File: code/game/gamemodes/malfunction/Malf_Modules.dm [hunk 1]
```diff
@@ -469,12 +469,12 @@
...
 	desc = "Overheats a machine, causing a small explosion after a short time."
 	button_icon_state = "overload_machine"
-	uses = 2
+	uses = 4
 	linked_ability_type = /obj/effect/proc_holder/ranged_ai/overload_machine
 
 /datum/action/innate/ai/ranged/overload_machine/proc/detonate_machine(obj/machinery/M)
 	if(M && !QDELETED(M))
-		explosion(get_turf(M), 0,1,1,0)
+		explosion(get_turf(M), 0,3,5,0)
```
Comment (Burzah) [inline code/game/gamemodes/malfunction/Malf_Modules.dm:477]:
```suggestion
		explosion(get_turf(M), 0, 3, 5, 0)
```

File: code/game/gamemodes/malfunction/Malf_Modules.dm [hunk 2]
```diff
@@ -469,12 +469,12 @@
 	name = "Overload Machine"
 	desc = "Overheats a machine, causing a small explosion after a short time."
```
Comment (Henri215) [inline code/game/gamemodes/malfunction/Malf_Modules.dm:470]: With the new explosion size, would you still consider it small or its better to change the description?

Comment (PopeDaveThe3th) [reply code/game/gamemodes/malfunction/Malf_Modules.dm:470]: Changed to "Moderately-sized".

Comment (Henri215) [inline code/game/gamemodes/malfunction/Malf_Modules.dm:470]:
```suggestion
	desc = "Overheats a machine, causing a moderately-sized explosion after a short time."
```

**Review by Henri215 — APPROVED** [review]

**Review by Burzah — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**